cores/arduino/am_sdk_ap3/mcu/apollo3/hal/am_hal_uart.h

```diff
@@ -172,8 +172,8 @@ am_hal_uart_transfer_t;
 
 // Flow control
 #define AM_HAL_UART_FLOW_CTRL_NONE          0
-#define AM_HAL_UART_FLOW_CTRL_CTS_ONLY      UART_CR_CTSEN_Msk
```

Comment:
Not sure why this is showing up in this PR... this change appears to already exist on release candidate.

Reply:
Sorry, Owen! I work in GitHub Desktop and I believe I may have mistakenly used "Merge into Current Branch" with the upstream/master and pushed the commit to the PR branch.